### DISCUSSION TITLE
Ошибка при заливке FS на ESP8266

### DISCUSSION THREAD

**Discussion by dimas4011** [vvip-68/LedPanelWiFi#16] [Вопросы и ответы]
Доброго времени суток!
Огромное спасибо за труд и терпение!)

Возникла проблемка с заливкой файловой системы на МК.
используется плата NodeMCU с чипом ESP-12F
Судя по описанию что-то не так с размером образа или непосредственно разбивкой памяти на МК.

> esptool.py v3.0
> Serial port COM3
> Connecting....
> Chip is ESP8285
> Features: WiFi, Embedded Flash
> Crystal is 26MHz
> MAC: 2c:f4:32:f4:2d:8c
> Uploading stub...
> Running stub...
> Stub running...
> Changing baud rate to 460800
> Changed.
> Configuring flash size...
> Auto-detected Flash size: 2MB
> 
> A fatal esptool.py error occurred: File C:\Users\Karlson\AppData\Local\Temp\arduino_build_798822/LedPanelWiFi_v1.14.mklittlefs.bin (length 2072576) at offset 2097152 will not fit in 2097152 bytes of flash. Use --flash-size argument, or change flashing address.

Плата NodeMCU V3 заливается на УРА и работает без проблем...

Прошу помощи  - куда копать? Программист из меня аховый конечно...

**Comment by vvip-68** [comment]
Что-то мне кажется на ESP 8285 это вообще работать не будет, тем более если у него размер памяти 2 Мегабайта.
В стандартном NodeMCU/Wemos на 8266 - там то 4 Мегабайта минимум...

**Reply by dimas4011** [reply]
Вона как... не увидал... врут значит гомосеки китайские.... 
На крышке написано ESP8266MOD

**Reply by vvip-68** [reply]
Ну это то, что пишет esptool.
Вполне возможно, что он использует информацию из настроек IDE, а в IDE у вас выбран не тот тип чипа. Тогда просто нужно выбрать правильную плату в менеджере плат. Для NodeMCU должно быть  "NodeMCU 1.0 (ESP12E Module)" Проверьте.

**Reply by dimas4011** [reply]
Да не, всё верно выставлено вроде...

![image](https://github.com/vvip-68/LedPanelWiFi/assets/148654532/efe139b5-b513-4778-b6fb-6984f87d7ca7)

**Reply by vvip-68** [reply]
Ну, тогда, видимо, увы...
А ссылку на лот продавца по этим платам можно? 
И фотку платы. Просто из любопытства...

**Reply by dimas4011** [reply]
Ссылка на лот:

https://aliexpress.ru/item/32665100123.html?spm=a2g2w.orderdetail.0.0.7c474aa6jb38fQ&sku_id=66676497214

Но там фотки отличаются, это по факту:

![image](https://github.com/vvip-68/LedPanelWiFi/assets/148654532/cb0af9a9-a9e2-4690-81d6-84a3928ca55c)

**Comment by vvip-68** [comment]
Странно всё это... На моей памяти китайцы на такой подмене не попадались... Выглядит вроде нормально. у 8285 даже корпус немного другой - уже вроде, судя по фоткам. Моя плата выглядит по факту так же...

**Reply by dimas4011** [reply]
Вот же и я о чём...
Задам вопрос китайцу, посмотрим
